docs/operations/export_api.md

```diff
@@ -0,0 +1,425 @@
+# Export API
+
+## Prerequesites
+None
+
+### Export Feature Gate
+
+VMExport support must be enabled in the feature gates to be supported. The
+[feature gates](./activating_feature_gates.md#how-to-activate-a-feature-gate)
+field in the KubeVirt CR must be expanded by adding the `VMExport` to it.
+
+### Export token
+
+In order to securely export a Virtual Machine Disk, you must create a token that is used to authorize users accessing the export endpoint. This token must be in a secret that is stored in the namespace the Virtual Machine resides in. The contents of the secret can be passed as a token header or parameter to the export URL. The name of the header or argument is `x-kubevirt-export-token` with a value that matches the content of the secret. The secret can be named any valid secret in the namespace. We recommend you generate an alpha numeric token of at least 12 characters. The data key should be `token`. For example:
```

Comment:
in the namespace where the Virtual Machine resides.

Reply:
Done